smart-contract-verifier/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM lukemathwalker/cargo-chef:0.1.39-rust-1.63-buster as chef
+WORKDIR /app
+
+FROM chef AS plan
+COPY . .
+RUN cargo chef prepare --recipe-path recipe.json
+
+FROM chef as cache
+COPY --from=plan /app/recipe.json recipe.json
+RUN cargo chef cook --release --recipe-path recipe.json
+
+FROM chef AS build
+
+COPY . .
+COPY --from=cache /app/target target
+COPY --from=cache $CARGO_HOME $CARGO_HOME
+RUN cargo build --release
+
+FROM ubuntu:20.04 as run
```

Comment:
Nit:

In cargo-chef description they use slim version of debian system in their example:
```
# We do not need the Rust toolchain to run the binary!
FROM debian:buster-slim AS runtime
```
Maybe we can also reuse that approach to decrease the size of our docker image?

Reply:
I tried to use `debian:buster-slim`, `distroless` and also several "small-weight" images, but vyper compiler always return error `lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.29' not found`. 

`debian:bullseye-slim` image worked, but it had the same image size as ubuntu, so I decided to leave ubuntu